modules/ROOT/pages/usage-metrics.adoc

```diff
@@ -104,30 +113,357 @@ Execution Type::
+
+Mule message processors in a Mule flow (such as core components, file read operations, or the HTTP request operations) can then retrieve, set, and process Mule message data that resides in the Mule event according to their configurations. 
+
+A Mule message is immutable, so every change to a Mule message results in the creation of a new instance. Each processor in a flow that receives a Mule message returns a new Mule message that consists of a message payload (the body of the message) and message attributes (metadata associated with the message).
+
+[[billable-mule-messages]]
+==== Mule Message Scenarios That Count Toward Your Anypoint Platform Package Allotment
+
+When an event source within a flow of a Mule application is triggered, the event source such as HTTP Listener or Scheduler, generates a Mule event that encapsulates a Mule message. The Mule message generated by the event source counts towards your Anypoint Platform package allotment. New instances of that message, which can be created during the processing of the original message as it moves through other processors in connected Mule flows, do not count towards your Anypoint Platform package allotment.
+
+[[data-throughput]]
+=== Total Data Throughput
+
+Data throughput is all of the network I/O bytes produced by the infrastructure that starts and runs the Mule Runtime server that runs a Mule application. This includes the data that the application produces to execute its business logic, as well as internal operational network traffic such as logs, health-checks, and monitoring traffic. For example, data throughput includes inserting a record into a database and the network traffic associated with the infrastructure of the app, such as log forwarding, control plane connection, and monitoring metrics transfer.  
+
+[[rtf-usage-metrics]]
+== Runtime Fabric Usage Metrics
+
+The Usage dashboard for Runtime Fabric lets you track, monitor, and govern core usage in hybrid environments and provides detailed metrics and analysis for Runtime Fabric instances and application deployments. 
+
+You can see your cluster capacity metrics refined by business group. To see CPU limit aggregates at the business group level, xref:usage-reports.adoc#export-usage-reports[export the usage report as a CSV]. The CPU limit is shown at the application level on the Usage dashboard, along with the associated business groups, which helps you identify which applications are using the most CPU, so you can make adjustments to your application configurations if needed. 
+
+Use the dashboard to compare cores and flows consumed by each application side-by-side to help you understand your utilization and associated pricing. You can visualize cores and flows consumed by each application alongside each other and drill down per environment ID and business group when you export the usage report CSV file. 
```

Comment:
In the dashboard, compare cores and flows consumed by each application to help you understand your usage and associated pricing. Export the usage report to a CSV file to visualize cores and flows consumed by each app and drill down per environment ID and business group. 
```